app-api/src/main/java/org/openmrs/eip/app/management/entity/DebeziumOffset.java

```diff
@@ -0,0 +1,65 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "debezium_offset")
+public class DebeziumOffset extends AbstractEntity {
+	
+	public static final long serialVersionUID = 1;
+	
+	@Column(nullable = false)
+	private byte[] data;
+	
+	@Column(name = "binlog_filename", nullable = false)
+	private String binlogFileName;
+	
+	@Column(name = "date_changed")
+	private LocalDateTime dateChanged;
+	
+	@Column(name = "enabled", nullable = false)
+	private Boolean enabled;
```

Comment:
What is the purpose of the enabled field? I hope it is not related to CustomFileOffsetBackingStore.disabled because I think we should store it in the DB

Reply:
No, they don't relate.

Following [this explanation](https://github.com/FriendsInGlobalHealth/openmrs-eip/pull/21#discussion_r1023979477), we need a way to discard the problematic offsets. 
We could either delete or deactivate. We chose the later one so the implementing team can have data to analyze the history in the future (which offsets was invalid, and the date were they was deactivated, and so on).
What are your thoughts?

Reply:
You should not save multiple offsets, save exactly one offset but first verify it before you save it

Reply:
I think that we are not in the same page. What kind of verification are you mentioning?

From our research, the main point here is that:
1. Debezium saves the offset
2. A power cut occurs
3. During the startup, the mysql does a crash recovery.
4. The last offset saved by the debezium engine can now point to an invalid position, due the previous crash recover. See [this](https://misfra.me/2017/08/02/recovering-mysql-replication-after-error-1236/) for more details.
5. We can either point to the first position from the next binlog file (which can lead to some missing events), or go back to a valid offset in the past.

So the client (debezium in this case) is the one that must adjust itself after the mysql crash recovery process ends, if I'm not missing something.

We can schedule a call to discuss this as a team.